ingestfile

```diff
@@ -482,6 +523,7 @@ if [ "${FROM}" == "${TO}" -a "${CLEANUP_CHOICE}" == "Y" ] ; then
         _run_critical mv -v -n "${FILES[@]}" "${LOGDIR}/"
     fi
 else
+    echo "BTUTUTUEUEUUUXUXU"
```

Comment:
Since there are so many shouts of BUUUUUUUUUUUUTS, BLURRRRRRRRRPS, and BTUTUTUEUEUUUXUXU, maybe this could be done in a function instead.

Reply:
:)